content/foundations/overview.html

```diff
@@ -1,6 +1,7 @@
 ---
 title: Foundations Overview
-description: A brand foundation is the core set of values and principles that define its identity and create a clear and consistent message..
+description: A brand foundation is the core set of values and principles that define its identity and create a clear and
+consistent message.
```

Comment:
Jekyll doesn't seem to load the page. Try putting all in one liner instead. Other than that, looks good.

Reply:
Ok